google/cloud/firestore_v1/services/firestore/transports/grpc_asyncio.py

```diff
@@ -215,12 +220,18 @@ def __init__(
                     scopes=self.AUTH_SCOPES, quota_project_id=quota_project_id
                 )
 
+            if client_cert_source_for_mtls and not ssl_channel_credentials:
```

Comment:
@busunkim96 do you know about the mtls bits here and if the veneers are expected to expose this?

Reply:
@arithmetic1728 has been going through and exposing the mTLS bits as different products GA mTLS support. I don't think there's action needed from the Firestore DPEs at the moment.

Reply:
Firestore is not on the GA list for mtls, please leave the bits as it is, we will revisit in the future to do necessary tweaks in the handwritten layer to make mtls work.